tests/test_build.py

```diff
@@ -78,10 +75,8 @@ def test_write(new_project_write):
     assert os.path.isfile(version_file)
 
     lines = read_file(version_file).splitlines()
-    if setuptools_scm_major < 7:
-        assert lines[3] == "version = '1.2.3'"
-    else:
-        assert lines[3] == "__version__ = version = '1.2.3'"
+    assert lines[3].startswith("version =") or lines[3].startswith("__version__ =")
```

Comment:
```suggestion
    assert lines[3].startswith(("version =", "__version__ =")) 
```

alltho im a bit torn bot whether thats a tad too compact

Reply:
Hmmm, I thought that this is not possible in Python 2 but it actually is - a tuple is a valid first argument here since Python 2.5.